.github/workflows/deploy_terraform.yml

```diff
@@ -51,7 +51,10 @@ jobs:
           sp-creds: ${{ secrets.SERVICE_PRINCIPAL_CREDS }}
           tf-auth: true
       - name: Collect Terraform stats
+
         uses: josiahsiegel/terraform-stats@68b8cbe42c494333fbf6f8d90ac86da1fb69dcc2
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/terraform-stats
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.